src/test/kotlin/com/terraformation/backend/customer/db/UserStoreTest.kt

```diff
@@ -773,7 +799,32 @@ internal class UserStoreTest : DatabaseTest(), RunsAsUser {
     fun `throws exception if user does not have a Terraformation email address`() {
       val userId = insertUser(10, email = "test@elsewhere.com")
 
-      assertThrows<AccessDeniedException> { userStore.updateGlobalRoles(userId, emptySet()) }
+      assertThrows<AccessDeniedException> { userStore.updateGlobalRoles(setOf(userId), emptySet()) }
```

Comment:
Can remove this test case since the newly-added one covers the behavior implicitly.